src/openssl/digest/digest_io.cr

```diff
@@ -0,0 +1,54 @@
+require "./digest_base"
+
+# Wraps an IO by calculating a specified digest on read or write operations.
```

Comment:
Please move the documentation block to the `DigestIO` class instead. This will avoid `OpenSSL` module receive the digest-related documentation.